### PR TITLE
Make checkmarks clickable

### LINKED ISSUE #137: Allow to click circles of checkmarks, not only text
Hi everyone!

A small usability idea: Currently when enabling options (e. g. Client Settings/Enable timer sounds) clicking the circle left of the option does not enable it, only clicking the text does.
I'm happy to provide a PR.

Best,
Daniel

### PR DESCRIPTION
**Pull request by dbartholomae**
This closes #137

### CODE REVIEW

File: public/components/checkbox.js [hunk 1]
```diff
@@ -19,19 +19,22 @@ export const checkbox = (props, children) => h('div', {
...
-  }, children),
+  }, [
+    h('span', {
+      class: {
+        'fa-stack': true,
+      },
+    }, [
+      h('i', { class: 'far fa-circle fa-stack-2x' }),
+      props.checked && h('i', { class: 'fas fa-check fa-stack-1x text-green-500' }),
+    ]), ...children]),
```
Comment (mrozbarry) [inline public/components/checkbox.js:39]: Everything looks good, and if you're alright with me nit-picking just a little, can you change this last part to:

```
  ]),
  ...children,
])
```

I found it just a smidge hard to follow with `...children` on the same line as a sibling.

**Review by mrozbarry — APPROVED** [review]
I'm going to approve, but if you want to fix my nit pick, I'll give you some time, and if you don't, I can resolve that.  In your defense, I don't have eslint/prettier set up to flag these things, so no biggy.

**Comment by mrozbarry** [comment]
Closes #137

File: public/components/checkbox.js (hunk 1 above)
Comment (dbartholomae) [reply public/components/checkbox.js:39]: Sure, will push in a couple minutes

**Comment by dbartholomae** [comment]
Changed the style. Please feel free to merge :)

**Comment by mrozbarry** [comment]
Wonderful, thanks friend!

**Comment by mrozbarry** [comment]
That is live now on https://dev.mobti.me - I'll probably glob a few updates together before it hits `www.` / `@.`, but you are free to use the dev version :smile:
